app/api/v1/antenna/validate.ts

```diff
@@ -1,32 +1,38 @@
 import { z } from 'zod';
 
 export type Antenna = {
-  id: string;
+  id: number;
```

Comment:
I think the type is the following:

```typescript
export type Antenna = {
  id: string;
  name: string;
  hostname: string;
  model: string;
  modelName: string;
  frequency: number;
  playground_frequency: number;
  latitude: string;
  longitude: string;
  azimuth: number;
  typeAntenna: 0 | 1 | 2;
  antenna_status: string;
  cpu: number | null;
  ram: number | null;
};```

Reply:
Actually not sure about cpu and ram. In the last meeting, we might have said to set it to 0

Reply:
@JonMike8 Can you change the schema to the one Owen provided please.